pydantic_core/_pydantic_core.pyi

```diff
@@ -53,14 +53,6 @@ class SchemaValidator:
         context: Any = None,
         self_instance: 'Any | None' = None,
     ) -> Any: ...
-    def isinstance_json(
-        self,
-        input: 'str | bytes | bytearray',
-        *,
-        strict: 'bool | None' = None,
-        context: Any = None,
-        self_instance: 'Any | None' = None,
-    ) -> bool: ...
     def validate_assignment(
```

Comment:
This isn't being used in pydantic, but I don't see `is_instance_python` being used anywhere in `pydantic` proper either. @samuelcolvin is this functionality that we just never ended up using, or am I missing something?